client/src/components/reviews/review_form/04.2_ShowRatingDesc.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+const ShowRatingDesc = (props) => {
+
+  const [showDesc, setShowDesc] = useState(false);
+
+  if (!showDesc) {
+    return (
+      <div className="caution-text" onClick={() => setShowDesc(current => !current)}> Show grading description </div>
+    );
+  } else {
+    return (
+      <article onClick={() => setShowDesc(current => !current)}>
+        Hide grading description
+        <ul className="characteristic-desc-list">
+          {props.desc[props.characteristic].map((desc, index) =>
+            <p key={index + 1} id={index + 1} > {`${index + 1} ${desc}`} </p>
```

Comment:
You shouldn't use the id attribute like this. The id attribute has to be unique for the entire page and should be used when an element itself is unique and needs to be addressed specifically for some reason. You could get away with using id here as long as you had the value be something like review-grading-description-x, but even then I would recommend against it unless you needed to denote this element uniquely (e.g. for styling purposes).

Reply:
Thanks @chefferson. You are right -- this `id` is not needed, and is removed now.